<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="Utama" native="true">
      <property name="styleSheet">
       <string notr="true">background-color:#DAD7CD;</string>
      </property>
      <widget class="QWidget" name="LeftMenu" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>131</width>
         <height>301</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#3A5A40;
border-radius:15px;</string>
       </property>
       <widget class="QPushButton" name="btnTraining">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#A3B18A;
text-weight: bold;
border-radius:10px;</string>
        </property>
        <property name="text">
         <string>Training</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../training.svg</normaloff>../training.svg</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="btnTesting">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#A3B18A;
text-weight: bold;
border-radius:10px;</string>
        </property>
        <property name="text">
         <string>Testing</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../testing.svg</normaloff>../testing.svg</iconset>
        </property>
       </widget>
       <widget class="QLabel" name="judul_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
</string>
        </property>
        <property name="text">
         <string>Menu</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="judul">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>20</y>
         <width>341</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SISTEM KLASIFIKASI SAMPAH</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>511</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnTraining_2">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>360</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#344E41;
text-weight: bold;
color :white;
border-radius:10px;</string>
       </property>
       <property name="text">
        <string>Training</string>
       </property>
      </widget>
      <widget class="QWidget" name="Utama_2" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>738</width>
         <height>466</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#DAD7CD;</string>
       </property>
       <widget class="QWidget" name="LeftMenu_2" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>131</width>
          <height>301</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#3A5A40;
border-radius:15px;</string>
        </property>
        <widget class="QPushButton" name="btnTraining_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#A3B18A;
text-weight: bold;
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>Training</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../training.svg</normaloff>../training.svg</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="btnTesting_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#A3B18A;
text-weight: bold;
border-radius:10px;</string>
         </property>
         <property name="text">
          <string>Testing</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../testing.svg</normaloff>../testing.svg</iconset>
         </property>
        </widget>
        <widget class="QLabel" name="judul_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>61</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;
</string>
         </property>
         <property name="text">
          <string>Menu</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="judul_4">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>20</y>
          <width>341</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>SISTEM KLASIFIKASI SAMPAH</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>90</y>
          <width>511</width>
          <height>251</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnTraining_4">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>360</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#344E41;
text-weight: bold;
color :white;
border-radius:10px;</string>
        </property>
        <property name="text">
         <string>Training</string>
        </property>
       </widget>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>360</y>
          <width>401</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#588157;
border-radius:5px;</string>
        </property>
        <widget class="QLabel" name="progres">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>5</y>
           <width>381</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="Class">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>400</y>
          <width>401</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
